src/panels/TcpDumpPanel.ts

```diff
@@ -1,21 +1,26 @@
 import { platform } from "os";
 import { relative } from "path";
 import { Uri, commands, window, workspace } from "vscode";
-import * as k8s from 'vscode-kubernetes-tools-api';
-import * as semver from 'semver';
+import * as k8s from "vscode-kubernetes-tools-api";
+import * as semver from "semver";
 import { failed, map as errmap, Errorable } from "../commands/utils/errorable";
 import { MessageHandler, MessageSink } from "../webview-contract/messaging";
 import { BasePanel, PanelDataProvider } from "./BasePanel";
 import { KubectlVersion, getExecOutput, invokeKubectlCommand } from "../commands/utils/kubectl";
-import { CompletedCapture, InitialState, ToVsCodeMsgDef, ToWebViewMsgDef } from "../webview-contract/webviewDefinitions/tcpDump";
+import {
+    CompletedCapture,
+    InitialState,
+    ToVsCodeMsgDef,
+    ToWebViewMsgDef,
+} from "../webview-contract/webviewDefinitions/tcpDump";
 import { withOptionalTempFile } from "../commands/utils/tempfile";
 
 const debugPodNamespace = "default";
 const tcpDumpCommandBase = "tcpdump --snapshot-length=0 -vvv";
 const captureDir = "/tmp";
 const captureFilePrefix = "vscodenodecap_";
 const captureFileBasePath = `${captureDir}/${captureFilePrefix}`;
-const captureFilePathRegex = `${captureFileBasePath.replace(/\//g, '\\$&')}(.*)\.cap`; // Matches the part of the filename after the prefix
+const captureFilePathRegex = `${captureFileBasePath.replace(/\//g, "\\$&")}(.*)\\.cap`; // Matches the part of the filename after the prefix
```

Comment:
Added a second `\` character to escape the other one, which would previously have been ignored, meaning the regex would match _any_ character before the extension, not just a literal `.`. So I think the linter caught a genuine bug here.